src/index.js

```diff
@@ -24,17 +24,13 @@ const slackBot = new SlackBot({
   token: process.env.SLACK_TOKEN,
```

Comment:
I tend to prefer to handle ENV vars first and then use the extracted values using local vars.  You could take advantage of the object destructuring to extract values:

```js
const {FIREBASE_PROJECT_ID, FIREBASE_CLIENT_EMAIL, FIREBASE_PRIVATE_KEY, FIREBASE_DB_URL, SLACK_TOKEN} = process.env;
```